renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": [
+    "config:base",
+    ":rebaseStalePrs",
+    ":preserveSemverRanges",
+    ":semanticPrefixChore",
+    ":disableRateLimiting"
```

Comment:
This keeps allows renovate to open as many update PRs as needed. (will be a lot at first)